apps/omg_child_chain/test/omg_child_chain/integration/fee_server_test.exs

```diff
@@ -136,9 +138,22 @@ defmodule OMG.ChildChain.Integration.FeeServerTest do
     pid = GenServer.whereis(TestFeeServer)
 
     capture_log(fn ->
-      Process.send(pid, :update_fee_spec, [])
-      # handle_info is async - we need to wait it executes to get message logged
-      Process.sleep(100)
+      logs = capture_log(fn -> Process.send(pid, :update_fee_spec, []) end)
+
+      case logs do
+        "" -> wait_for_log()
+        logs -> logs
+      end
+    end)
+  end
+
+  defp wait_for_log do
```

Comment:
Thinking of yours `call_until` as a test level helper function. This log handling case should be tricky enough that we could end with nice generic function that handles all wait_for cases.